srcpkgs/spice/template

```diff
@@ -10,17 +10,23 @@ makedepends="spice-protocol libjpeg-turbo-devel pixman-devel
  openssl-devel opus-devel alsa-lib-devel libXfixes-devel libXrender-devel
  libXrandr-devel libsasl-devel libXinerama-devel libglib-devel libcacard-devel
  nss-devel"
+checkdepends="glib-networking"
 short_desc="Implements the SPICE protocol"
 maintainer="Anachron <gith@cron.world>"
 license="LGPL-2.1-or-later"
 homepage="http://www.spice-space.org"
-distfiles="http://www.spice-space.org/download/releases/spice-${version}.tar.bz2"
-checksum=551d4be4a07667cf0543f3c895beb6da8a93ef5a9829f2ae47817be5e616a114
+distfiles="https://www.spice-space.org/download/releases/spice-server/spice-${version}.tar.bz2"
+checksum=b320cf8f4bd2852750acb703c15b72856027e5a8554f8217dfbb3cc09deba0f5
 
 if [ "$XBPS_TARGET_ENDIAN" != "le" ]; then
 	broken="SPICE server only works on little endian architectures"
 fi
 
+if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
+  # tests depend on odd behaviour of glibc
+  make_check=no
+fi
```

Comment:
Please document what the odd behavior is. If it gets too long, it can happen in the commit message; it should at least be tracked.

Reply:
is this better?